### ISSUE TITLE
[10.0][sythil_saas_server_domain_filter] Error when trying to install Odoo 10 in new server

### ISSUE THREAD

**Issue by Olageibol** [SythilTech/Odoo10#25]
I'm getting the following error while trying to install odoo 10:

2017-10-17 15:58:30,828 7021 ERROR ? werkzeug: Error on request:
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 209, in run_wsgi
    execute(self.server.app)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/serving.py", line 197, in execute
    application_iter = app(environ, start_response)
  File "/opt/odoo/repos/OCB/odoo/service/server.py", line 249, in app
    return self.app(e, s)
  File "/opt/odoo/repos/OCB/odoo/service/wsgi_server.py", line 186, in application
    return application_unproxied(environ, start_response)
  File "/opt/odoo/repos/OCB/odoo/service/wsgi_server.py", line 172, in application_unproxied
    result = handler(environ, start_response)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1308, in __call__
    return self.dispatch(environ, start_response)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1282, in __call__
    return self.app(environ, start_wrapped)
  File "/usr/local/lib/python2.7/dist-packages/werkzeug/wsgi.py", line 600, in __call__
    return self.app(environ, start_response)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1446, in dispatch
    self.setup_db(httprequest)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1368, in setup_db
    httprequest.session.db = db_monodb(httprequest)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1530, in db_monodb
    dbs = db_list(True, httprequest)
  File "/opt/odoo/repos/OCB/odoo/http.py", line 1499, in db_list
    return db_filter(dbs, httprequest=httprequest)
  **File "/opt/odoo/repos/sythiltech/sythil_saas_server_domain_filter/models/http.py", line 22, in db_filter
    saas_server_db = odoo.tools.config['sythilsaasserver']**
  File "/opt/odoo/repos/OCB/odoo/tools/config.py", line 579, in __getitem__
    return self.options[key]
KeyError: 'sythilsaasserver'

**Comment by SythilTech** [comment]
Hello,

Please ensure you have followed the documentation here
https://github.com/SythilTech/Odoo10/blob/10.0/sythil_saas_server_domain_filter/static/description/index.html

In particular it talks about editing your odoo/tools.config.py file to include the name of the database which has the saas server installed..

<pre>group.add_option("--sythilsaasserver", dest="sythilsaasserver", my_default='saas', help="The name of the saas server database")</pre>

Basically the module works by quickly connecting to the saas database, looking up which database uses the current url then doing a db_filter to only that database.

Regards
Steven
